include/mbedtls/md.h

```diff
@@ -63,65 +65,69 @@ typedef enum {
 #endif
 
 /**
- * Opaque struct defined in md_internal.h
+ * Opaque struct defined in md_internal.h.
  */
 typedef struct mbedtls_md_info_t mbedtls_md_info_t;
 
 /**
- * Generic message digest context.
+ * The generic message digest context.
```

Comment:
The spelling of `message digest` vs. `message-digest` should be uniform. Above, it's now including the hyphen.

Reply:
It needed correcting in some places, but not all. The logic is:
If I am just talking about MD, it is: message digest.
If I am using message digest as a type of something else: message-digest information, message-digest context, etc.
I hope that is clear. Made consistent per the above classificaftions.